sematic/ui/src/components/LogPanel.tsx

```diff
@@ -37,7 +41,7 @@ export default function LogPanel(props: { run: Run }) {
       {run.exception_metadata_json && <Box sx={{ paddingBottom: 4, }} >
         <Exception exception_metadata={run.exception_metadata_json} /></Box>}
 
-      <Box sx={{ flexGrow: 1, display: 'flex', flexDirection: 'column'}} >
+      {!runHasException && <Box sx={{ flexGrow: 1, display: 'flex', flexDirection: 'column'}} >
```

Comment:
Is this properly aligned?

Reply:
Yes. I think so. Similar to the format on line 41 as well.